apps/web-staking/src/app/components/links/LinkComponent.tsx

```diff
@@ -1,7 +1,16 @@
 "use client";
 
 import Link from "next/link";
-import { ElementType } from "react";
+import { ElementType, ReactElement, SVGProps } from "react";
+
+// Define a custom type for the Icon component
+type IconProps = {
+  width?: number;
+  height?: number;
+  fill?: string;
+};
+
+export type IconComponent = (props: IconProps) => ReactElement;
```

Comment:
Had to do this to fix a new build error for types.

![Error](https://github.com/user-attachments/assets/e4b9c89c-8a10-4c6f-a4a9-d196c6c62bc4)